dashboard/app/views/levels/editors/_gamelab.html.haml

```diff
@@ -28,6 +28,21 @@
   = boolean_check_box f, :show_d_pad
 
 .field
+  = f.label :show_debug_watch, 'Show Watchers UI in Debug Area'
+  = boolean_check_box f, :show_debug_watch
+
+.field
+  = f.label 'Pre-populated Watchers'
+  %pre
+    = preserve do
+      :escaped
+        Example:
+        ["Game.mouseX", "Game.mouseY", "Game.width"]
+  = f.text_area 'watchers_prepopulated', placeholder: 'Insert JSON array', rows: 4, value: @level.watchers_prepopulated or ''
+  :javascript
+    levelbuilder.initializeCodeMirror('level_watchers_prepopulated', 'javascript');
```

Comment:
Since this is repeated between two editors, should we instead implement it as a partial that gets included here (like [levels/editors/_encrypted_examples.html.haml](https://github.com/code-dot-org/code-dot-org/blob/5b79673e7b90bef9fa2c2b12bb1811ea94374a7b/dashboard/app/views/levels/editors/_encrypted_examples.html.haml#L1-L1))?